extension/icu/icu_extension.cpp

```diff
@@ -68,18 +83,27 @@ struct IcuBindData : public FunctionData {
 		auto &bind_data = bind_data_p->Cast<IcuBindData>();
 		serializer.WriteProperty(100, "language", bind_data.language);
 		serializer.WriteProperty(101, "country", bind_data.country);
+		serializer.WriteProperty(102, "tag", bind_data.tag);
```

Comment:
```suggestion
		serializer.WritePropertyWithDefault(102, "tag", bind_data.tag,"");
```

Reply:
Do you think the default value is also required for the other fields: language and country?
It could be the case that a tagged collation is supplied and those fields might be empty.

Reply:
I think ONLY tag should have `WithDefault`. This has to do with serialization logic, WithDefault implies that it would be serialized as having only 2 fields (= compatible with DB files created with upcoming v0.10.3) IFF tag would not be present, and only when present the deserializer would recognize (and handle) the problem.